src/frollR.c

```diff
@@ -113,7 +113,7 @@ SEXP frollfunR(SEXP fun, SEXP obj, SEXP k, SEXP fill, SEXP algo, SEXP align, SEX
   if (!dans) error("%s: Unable to allocate memory answer", __func__); // # nocov
   double* dx[nx];                                               // pointers to source columns
   uint_fast64_t inx[nx];                                        // to not recalculate `length(x[[i]])` we store it in extra array
-  if (bverbose) Rprintf("%s: allocating memory for results %dx%d\n", __func__, nx, nk);
+  if (verbose) Rprintf("%s: allocating memory for results %dx%d\n", __func__, nx, nk);
```

Comment:
`bverbose` was correct here.  `verbose` is the SEXP passed to the function.  I've often appended "Arg" to argument names at C level to avoid confusion like this, since we often use the SEXP args at the top of the function to check and extract the raw C values (like bool verbose, called `bverbose` in this case). The change to `verbose` is passing checks because it's always true;  a pointer value is always true (the argument is never NULL).  The extra verbosity is passing checks because our `test()` doesn't check for the lack of verbosity; that's usually caught when we or users run test.data.table ourselves.   But that doesn't catch this one either because this one is over in the separate froll tests script.